homeassistant/components/devolo_home_network/coordinator.py

```diff
@@ -0,0 +1,42 @@
+"""Base coordinator."""
+
+from asyncio import Semaphore
+from collections.abc import Awaitable, Callable
+from datetime import timedelta
+from logging import Logger
+
+from typing_extensions import TypeVar
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+_DataT = TypeVar("_DataT")
+
+
+class DevoloDataUpdateCoordinator(DataUpdateCoordinator[_DataT]):
```

Comment:
```suggestion
class DevoloDataUpdateCoordinator[_DataT](DataUpdateCoordinator[_DataT]):
```
I think you can use the new notation of types